entrypoint_scripts/common/common-os.sh

```diff
@@ -115,6 +115,15 @@ bootstrap_install() {
 	    echo "Bootstapping CentOS"
 	    echo "  TBD: Pre-reqs for CentOS"
 	    ;;
+	    "Linux Mint")
```

Comment:
As Ubuntu and Linux Mint are handled the same way here, this structure creates code duplication, making it harder for maintenance.
```suggestion
	    "Ubuntu" | "Linux Mint")
```